jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/ServletContextRequest.java

```diff
@@ -304,6 +305,9 @@ public Charset getQueryEncoding()
     @Override
     public Object getAttribute(String name)
     {
+        if (ServletContext.TEMPDIR.equals(name))
+            return getContext().getTempDirectory();
```

Comment:
This behaves kinda like a synthetic attribute now.

Note: the `ServletContext.TEMPDIR` is a ServletContext attribute, not a Request attribute (I think this code is in the wrong place)

Will this attribute show up when a user iterates the attributes now?

Reply:
You're right, let me fix that.